rdr_service/config/pub_sub_config_stable.json

```diff
@@ -0,0 +1,235 @@
+      "payload_format": "JSON_API_V1",
+      "topic_name": "genomic_manifest_upload",
+      "topic_project": "all-of-us-rdr-prod"
+    },
+    {
+      "bucket": "genomics-raw-color",
+      "event_types": [
+        "OBJECT_FINALIZE"
+      ],
+      "id": "4",
+      "object_name_prefix": "W3SC_manifests/",
+      "payload_format": "JSON_API_V1",
+      "topic_name": "genomic_manifest_upload",
+      "topic_project": "all-of-us-rdr-prod"
+    },
+    {
+      "bucket": "genomics-raw-color",
+      "event_types": [
+        "OBJECT_FINALIZE"
+      ],
+      "id": "5",
+      "object_name_prefix": "W4WR_manifests_hdr/",
+      "payload_format": "JSON_API_V1",
+      "topic_name": "genomic_manifest_upload",
+      "topic_project": "all-of-us-rdr-prod"
+    },
+    {
+      "bucket": "genomics-raw-color",
+      "event_types": [
+        "OBJECT_FINALIZE"
+      ],
+      "id": "6",
+      "object_name_prefix": "W4WR_manifests_pgx/",
+      "payload_format": "JSON_API_V1",
+      "topic_name": "genomic_manifest_upload",
+      "topic_project": "all-of-us-rdr-prod"
+    },
+    {
+      "bucket": "genomics-raw-color",
+      "event_types": [
+        "OBJECT_FINALIZE"
+      ],
+      "id": "7",
+      "object_name_prefix": "W5NF_manifests/",
+      "payload_format": "JSON_API_V1",
+      "topic_name": "genomic_manifest_upload",
+      "topic_project": "all-of-us-rdr-prod"
+    },
+    {
+      "bucket": "genomics-raw-northwest",
+      "event_types": [
+        "OBJECT_FINALIZE"
+      ],
+      "id": "17",
+      "object_name_prefix": "W5NF_manifests/",
+      "payload_format": "JSON_API_V1",
+      "topic_name": "genomic_manifest_upload",
+      "topic_project": "all-of-us-rdr-prod"
+    },
+    {
+      "bucket": "genomics-raw-northwest",
+      "event_types": [
+        "OBJECT_FINALIZE"
+      ],
+      "id": "18",
+      "object_name_prefix": "W4WR_manifests_hdr/",
+      "payload_format": "JSON_API_V1",
+      "topic_name": "genomic_manifest_upload",
+      "topic_project": "all-of-us-rdr-prod"
+    },
+    {
+      "bucket": "genomics-raw-northwest",
+      "event_types": [
+        "OBJECT_FINALIZE"
+      ],
+      "id": "19",
+      "object_name_prefix": "W4WR_manifests_pgx/",
+      "payload_format": "JSON_API_V1",
+      "topic_name": "genomic_manifest_upload",
+      "topic_project": "all-of-us-rdr-prod"
+    },
+    {
+      "bucket": "genomics-raw-northwest",
+      "event_types": [
+        "OBJECT_FINALIZE"
+      ],
+      "id": "25",
+      "object_name_prefix": "W2SC_manifests/",
+      "payload_format": "JSON_API_V1",
+      "topic_name": "genomic_manifest_upload",
+      "topic_project": "all-of-us-rdr-prod"
+    },
+    {
+      "bucket": "genomics-raw-northwest",
+      "event_types": [
+        "OBJECT_FINALIZE"
+      ],
+      "id": "26",
+      "object_name_prefix": "W3SC_manifests/",
+      "payload_format": "JSON_API_V1",
+      "topic_name": "genomic_manifest_upload",
+      "topic_project": "all-of-us-rdr-prod"
```

Comment:
I'm not sure how the `topic_project` is used, but just wanted to confirm that the Stable JSON pub-sub config entries should have the topic_project set to "all-of-us-rdr-prod".

Reply:
Bad copypasta...

https://github.com/all-of-us/raw-data-repository/pull/2939/commits/0d7f89824fbd107554247bb3c6dd3fd7d4b01d77